api/internal/owner/serializers.py

```diff
@@ -246,18 +246,20 @@ class RootOrganizationSerializer(serializers.Serializer):
 
 
 class AccountDetailsSerializer(serializers.ModelSerializer):
+    # circular imports
+    from api.internal.account.serializers import BaseAccountSerializer
```

Comment:
This leads to circular imports? 💀

Reply:
yeah, the location and naming of serializers is a bit of a mess, but the `PlanSerializer` is defined here (in the `Owners` file 💀 ), and I want to use it on `BaseAccountSerializer` (it's important that the Account `plan` is serialized the same as the Org `plan` so that one can replace the other), which gets circular since I want account on `AccountDetailsSerializer` (which is actually a `ModelSerializer` for `Owners` 💀 💀 💀 )